R/snapshot.R

```diff
@@ -890,17 +882,26 @@ renv_snapshot_filter_packages <- function(project, records, packages) {
   paths <- renv_package_dependencies(packages, project = project)
   all <- as.character(names(paths))
   kept <- keep(records, all)
+  kept <- renv_add_bioc_packages(kept, records, project)
 
+  kept
+
+}
+
+renv_add_bioc_packages <- function(packages, records, project = NULL){
```

Comment:
Can you please rename this to e.g. `renv_bioconductor_packages` or something similar? (Anything relevant to Bioconductor functionality should normally be prefixed with `renv_bioconductor_`)

Reply:
(Or, if you're keeping this in `snapshot.R` then `renv_snapshot_bioconductor_packages`)